<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitiationQtClass</class>
 <widget class="QMainWindow" name="InitiationQtClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InitiationQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>38</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>68</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>98</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>status de connexion : deconnecte</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessageEditLine">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PS : 127.0.0.1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Message">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnC">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnF">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>°F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnHr">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisissez le capteur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>type de donnee : 01</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InitiationQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnC</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>onSendCelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnF</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>onSendFarClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnHr</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>onSendHygClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDisplayMessageButtonClicked()</slot>
  <slot>onConnectButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onServerNewConnection()</slot>
  <slot>onSendCelClicked()</slot>
  <slot>onSendFarClicked()</slot>
  <slot>onSendHygClicked()</slot>
 </slots>
</ui>
